docs/mkdocs.yml

```diff
@@ -136,7 +136,6 @@ nav:
       - guides/fabric/upgrade-network-1.4.x-2.2.x.md
       - guides/fabric/upgrade-network.md
     - Corda Operations:
-      - guides/corda/add-cenm-console.md
```

Comment:
Is this feature not covered anymore?

Reply:
the feature is still there, but there is no need to add/enable cenm-console, it is by default